src/test/eo/org/eolang/collections/hash-code-of-tests.eo

```diff
@@ -42,30 +39,132 @@
           hash-code-of FALSE
     $.equal-to TRUE
 
-[] > hash-of-int-is-int
+[] > hash-code-of-int-is-int
   assert-that > @
     is-int.
       number
         hash-code-of 42
     $.equal-to TRUE
 
-[] > hash-codes-of-the-same-numbers-are-equal
+[] > hash-codes-of-the-same-int-are-equal-1
+  123456789012345678 > big-int!
+  assert-that > @
+    hash-code-of big-int
+    $.equal-to
+      hash-code-of big-int
+
+[] > hash-codes-of-the-same-int-are-equal-2
```

Comment:
@c71n93 it would be better to remove digits from the name of the tests here and below